### ISSUE TITLE
[QST] Do we support row-based file compress/decompress ? 

### ISSUE THREAD

**Issue by chenrui17** [NVIDIA/nvcomp#46]
Such as  a txt file which is compressed by zlib ,and i want to decompress it with gpu , can i finish it use nvcom ?

**Comment by mnicely** [comment]
Just to make sure we understand. Are you asking if you can compress using `zlib` on CPU and decompress using `nvCOMP` on GPU?

**Comment by chenrui17** [comment]
> Just to make sure we understand. Are you asking if you can compress using `zlib` on CPU and decompress using `nvCOMP` on GPU?

yes, it seems that we cannot speed up decompress an entire large gzip file on GPU which is compressed used `zlib` in cpu.
